mhtc-web/components/GoogleSearchConsole.tsx

```
@@ -0,0 +1,64 @@
+export function GoogleSearchConsole({ verificationCode }: GoogleSearchConsoleProps) {
+  // You can replace this with your actual verification code from Google Search Console
+  const defaultVerificationCode = 'REPLACE_WITH_YOUR_GOOGLE_VERIFICATION_CODE'
+  const finalVerificationCode = verificationCode || defaultVerificationCode
+
+  return (
+    <>
+      {/* Google Search Console Verification Script */}
+      <Script
+        id="google-search-console-verification"
+        strategy="afterInteractive"
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)